pkg/parsing/impl/validator.go

```diff
@@ -62,33 +61,24 @@ func (pp *QueryValidator) ValidateCreateTable(query string, chainID tableland.Ch
 	}
 
 	node := stmt.(*sqlparser.CreateTable)
-	if !pp.createTableNameRegEx.MatchString(node.Table.String()) {
-		return nil, &parsing.ErrInvalidTableName{}
-	}
-	parts := strings.Split(node.Table.String(), "_")
-	if len(parts) < 2 {
-		return nil, fmt.Errorf("table name isn't referencing the chain id")
+	validTable, err := sqlparser.ValidateCreateTargetTable(node.Table)
```

Comment:
Using the parser package method to validate the table of a CREATE statement

Reply:
Most of the changes in this file is do adjust the code to the fact that the validation methods return the types `ValidatedTable` and `ValidatedCreateTable`